doc/05-component-kernel-implementation/05-component-kernel-implementation.md

```diff
@@ -111,9 +109,6 @@ do good work.
 > representation now and justify it. Feel free to also use this space to
 > discuss how that representation will be restricted (i.e., by convention)
 > and interpreted (i.e., by correspondence).
```

Comment:
Make sure you argue why you selected the representation you did.

Reply:
I have added a justification in Chord1L above the instantiation of my rep.